brian2/__init__.py

```diff
@@ -2,10 +2,9 @@
 Brian 2
 '''
 # Import setuptools to do some monkey patching of distutils, necessary for
-# working weave/Cython on Windows with the Python for C++ compiler
-from __future__ import absolute_import
+# working Cython on Windows with the Python for C++ compiler
+
 import setuptools as _setuptools
```

Comment:
Ok, I've changed it to simply `import setuptools` now, but should it be removed entirely?

Reply:
I think it can be removed completely. Importing it meant that it monkey-patched distutils and the standard extension compilation process (used by weave and Cython) would therefore pick up the C++ compiler for Python 2.7 correctly.

Reply:
Done!